Add WISE datasets

troyraen opened the pull request:
Adds
- allsky
- cryo-3band
- postcryo
- neowiser

Updates
- allwise

## To Do

- [x] add dataset and resource descriptions
- [x] add DOIs
- [x] get reviews from ~Vandana~ David and Brigitta
- [x] submit PR to awslabs/open-data-registry
- [x] submit an application for each dataset at https://application.opendata.aws/ (for dataset tracking purposes only)
- [x] send logo to opendata@amazon.com

damercha committed:
Merge branch 'main' of https://github.com/awslabs/open-data-registry into awslabs-main

damercha committed:
Merge branch 'awslabs-main'

damercha committed:
Update amazon-last-mile-challenges.yaml

Updates to description, documentation and citation.

damercha committed:
Merge branch 'awslabs:main' into main

damercha committed:
Update amazon-last-mile-challenges.yaml

Fixed typo

damercha committed:
Merge branch 'main' of https://github.com/damercha/open-data-registry

damercha committed:
Update amazon-last-mile-challenges.yaml

Replaced data set with dataset

damercha committed:
Merge branch 'main' of https://github.com/awslabs/open-data-registry into main

damercha committed:
Update amazon-last-mile-challenges.yaml

Fixed typos

troyraen committed:
condense multi-line strings to single line.
they are causing spacing problems on the rendered page.
fields are rendered as single paragraphs.

troyraen committed:
update ManagedBy

troyraen committed:
fix url

troyraen committed:
Merge branch 'awslabs:main' into update/allwise

troyraen committed:
add allsky

troyraen committed:
add cryo-3band

troyraen committed:
add neowiser

troyraen committed:
add postcryo

troyraen committed:
update wise datasets

troyraen commented on datasets/wise-neowiser.yaml:
```diff
@@ -0,0 +1,29 @@
+License: https://irsa.ipac.caltech.edu/data_use_terms.html
+Citation: "If you use the NEOWISE-R Single-exposure Image Sets, please cite the Digital Object Identifier (DOI): https://www.ipac.caltech.edu/doi/irsa/10.26131/IRSA147. If you use the NEOWISE-R Single-exposure Source Database, please cite the Digital Object Identifier (DOI): https://www.ipac.caltech.edu/doi/irsa/10.26131/IRSA144. In addition, follow the [NEOWISE acknowledgement guidelines](https: ...
+Resources:
+  - Description: The Single-exposure Image Sets consist of more than 20 million calibrated 1016x1016 pix @2.75"/pix FITS image sets for the individual 7.7 sec W1 and W2 NEOWISE survey exposures. Each image set consists of two intensity images, noise maps, and bit-masks indicating pixel use status, one each for the W1 and W2 bands. The Single-exposure Source Database contains time-tagged fluxes, p ...
```
The Single-exposure Source Database (parquet catalogs for years 1-9) is not in the bucket yet. The files have been created and verified but the ancillary files need to be created before uploading. I'm not sure how we feel about submitting the yaml before uploading those.

stargaser replied:
I suppose the Parquet files are so close that it won't cause much harm to submit the yaml file now.

troyraen commented:
@bsipocz will you give these a sanity check? Same idea as the recent spitzer-seip one. Vandana is in the process of reviewing them as well. This is just in my fork for now; I'll submit to the upstream repo after we work out the kinks.

bsipocz commented on datasets/wise-allsky.yaml:
```diff
@@ -0,0 +1,28 @@
+Tags:
+  - astronomy
+  - imaging
+  - satellite imagery
```
I'm uncertain this is a correct tag for this type of dataset, while the data is taken from a satellite, I think this one is for those ones looking down :)
(comment stands for all the other usage of this tag, too)

bsipocz replied:
(not sure, is there maybe a wavelength-dependent tag? e.g. infrared imaging could be one)

troyraen replied:
Thanks for taking time to look at this. I checked with Vandana about the satellite imagery tag after you commented on the spitzer-seip submission and we opted to keep it. It is ambiguous and AWS gives zero guidance about what the intended meaning of the tags are. So we opted for more tags to be more descriptive. I like the "infrared imaging" suggestion. There's nothing like that in the current tags (although there are x-ray ones) but we can propose new tags by submitting a PR. I'll check with Vandana if she wants me to do that.

bsipocz reviewed (commented):
All look good, my only comment is about the tags.

troyraen commented:
@stargaser

Will you review this? Here’s some additional info, if it’s helpful:

- These yaml files will be used to display a webpage for each dataset. Here is the current [AllWISE page](https://registry.opendata.aws/wise-allwise/) for reference.
- The bucket/folder containing the data described in these yamls: [nasa-irsa-wise/wise](https://s3.console.aws.amazon.com/s3/buckets/nasa-irsa-wise?prefix=wise/&region=us-west-2).
- Yaml field descriptions are in the upstream [repo readme](https://github.com/awslabs/open-data-registry).

cronosnull committed:
[avianmonitoring] Gulfwide Avian Colony Monitoring Survey Photos

stargaser commented on datasets/wise-allwise.yaml:
```diff
@@ -1,28 +1,20 @@
 Documentation: https://irsa.ipac.caltech.edu/Missions/wise.html
 Contact: https://irsa.ipac.caltech.edu/docs/help_desk.html
-ManagedBy: "[Caltech IPAC-IRSA](https://irsa.ipac.caltech.edu)"
+ManagedBy: "NASA/IPAC Infrared Science Archive ([IRSA](https://irsa.ipac.caltech.edu)) at Caltech"
```
Will the line that is now "See all datasets managed by [Caltech IPAC-IRSA](https://registry.opendata.aws/?search=managedBy:caltech%20ipac-irsa)" become "See all datasets managed by [NASA/IPAC Infrared Science Archive (IRSA) at Caltech](https://registry.opendata.aws/?search=managedBy:nasa/ipac%20infrared%20science%20archive%20(IRSA)%20at%20caltech)"? I suppose it will still work.

troyraen replied:
We decided to go with the longer name when we submitted [spitzer-seip](https://registry.opendata.aws/spitzer-seip/). So it should look the same as that one, but... I expected that one to have our logo and then say:

> See all datasets managed by NASA/IPAC Infrared Science Archive ([IRSA](https://irsa.ipac.caltech.edu/)) at Caltech

But the Spitzer page doesn't have our logo yet and instead says:

> NASA/IPAC Infrared Science Archive ([IRSA](https://irsa.ipac.caltech.edu/)) at Caltech
> 
> See all datasets managed by [IRSA](https://registry.opendata.aws/?search=managedBy:irsa).

So I'll have to follow up there.

troyraen replied:
Thanks for taking the time to look through these.

stargaser commented on datasets/wise-allsky.yaml:
```diff
@@ -0,0 +1,28 @@
+License: https://irsa.ipac.caltech.edu/data_use_terms.html
+Citation: "If you use the All-Sky Single-exposure Image Sets, please cite the Digital Object Identifier (DOI): https://www.ipac.caltech.edu/doi/irsa/10.26131/IRSA152. In addition, follow the [All-Sky acknowledgement guidelines](https://wise2.ipac.caltech.edu/docs/release/allsky/expsup/sec1_6b.html) and the [IRSA acknowledgement guidelines](https://irsa.ipac.caltech.edu/ack.html)."
+Resources:
+  - Description: All-Sky Single-exposure Image Sets
```
The Description section appears twice in these yaml files and I see from the AllWISE entry that the Resources:Description includes all of the first Description, plus a bit more about the partitioning of the Parquet files. For the image-only sets, here and elsewhere, maybe this could be expanded to

All-Sky Single-exposure Image Sets. Each image set consists of four intensity images in FITS format, noise maps, and bit-masks indicating pixel use status, one for each WISE band.

troyraen replied:
Ya, more than one resource can be registered for the same dataset, but ours only need the one. Since the two descriptions sit next to each other on the webpage it seems unnecessary to repeat it in both places (though I didn't change AllWISE here, just fixed the formatting). But you're right, a little more info in the resource-level description might be good. I'll add what you suggested.

stargaser reviewed (commented):
Looks good overall to me, I just had a couple of inline comments.

troyraen committed:
expand descriptions

troyraen committed:
update dataset name

troyraen committed:
Merge branch 'awslabs:main' into add/wise

troyraen committed:
quote strings with special characters

Patrick-Keown committed:
Update nwm-archive.yaml

Updated browse bucket links

cstner committed:
Merge pull request #2027 from Patrick-Keown/patch-154

Update nwm-archive.yaml

cronosnull committed:
updating tags

cronosnull committed:
adding aws-pds

cronosnull committed:
fixing link

cronosnull committed:
fix yaml syntax

cronosnull committed:
Merge branch 'main' into avianmonitoring

cstner committed:
Merge pull request #2021 from waterinstitute/avianmonitoring

[avianmonitoring] Gulfwide Avian Colony Monitoring Survey Photos

matthew-mizielinski committed:
Add yaml file for Met Office UKESM1 ARISE-SAI

Add yaml file for Met Office UKESM1 ARISE-SAI datasets

matthew-mizielinski committed:
Update and rename met-office-ukesm1-arise.yml to met-office-ukesm1-arise.yaml

Correct file name

matthew-mizielinski committed:
Fix Typo in resource type

matthew-mizielinski committed:
Update author name and remove other blank fields

cstner committed:
Merge pull request #2028 from matthew-mizielinski/main

Add yaml file for Met Office UKESM1 ARISE-SAI

uscbdatareleases committed:
Add files via upload

uscbdatareleases committed:
Updated description

matthew-mizielinski committed:
Add explore link to point at the bucket index page

cstner committed:
Merge pull request #2032 from matthew-mizielinski/main

met-office-ukesm1-arise: add Browse Bucket link

uscbdatareleases committed:
Merge pull request #4 from uscbdatareleases/uscbdatareleases-patch-3

Updated description

uscbdatareleases committed:
Merge branch 'awslabs:main' into main

lhuang83 committed:
Update publications in wrf-cmip6.yaml

NiuZhixiao committed:
Update cmip6-ers5-hybrid-southeast-asia.yaml

briandobbins committed:
Added 'Explore' entry w/ URL

pschmied committed:
Merge pull request #2030 from uscbdatareleases/main

Created 2020 DHC NMF

lhuang83 committed:
Merge branch 'main' into patch-7

uscbdatareleases committed:
Add files via upload

pschmied committed:
Merge pull request #2037 from uscbdatareleases/main

Updated URL in the description

khawki29 committed:
add dataset for UCSF-JHU Opioid Industry Documents Archive

khawki29 committed:
commented out DataAtWork section

Documentation said that this section is optional, but maybe you need to comment it out to pass validation

Patrick-Keown committed:
Update noaa-ndfd.yaml

Added Public Notice Comment Period information for NDFD

lhuang83 committed:
Merge branch 'main' into patch-7

augustus-thomas committed:
Update nasa-power.yaml misspelling

The word "variety" was written "varity" in the description of this dataset. It bothered me so I forked and fixed it for you

gabriellaIO committed:
Update io-lulc.yaml

Stac endpoint link was not working. Fixed

pschmied committed:
Merge pull request #2041 from augustus-thomas/patch-1

Update nasa-power.yaml misspelling

pschmied committed:
Merge branch 'main' into patch-155

pschmied committed:
Merge branch 'main' into main

pschmied committed:
Merge pull request #2038 from khawki29/main

add dataset for UCSF-JHU Opioid Industry Documents Archive

pschmied committed:
Merge branch 'main' into patch-155

pschmied committed:
Merge pull request #2039 from Patrick-Keown/patch-155

Update noaa-ndfd.yaml

pschmied committed:
Merge branch 'main' into patch-1

pschmied committed:
Update cmip6-ers5-hybrid-southeast-asia.yaml

Typo

pschmied committed:
Merge branch 'main' into patch-7

pschmied committed:
Merge pull request #2033 from lhuang83/patch-7

Update publications in wrf-cmip6.yaml

pschmied committed:
Merge branch 'main' into patch-1

pschmied committed:
Update and rename cmip6-ers5-hybrid-southeast-asia.yaml to cmip6-era5-hybrid-southeast-asia.yaml

pschmied committed:
Merge pull request #2035 from NiuZhixiao/patch-1

Update cmip6-ers5-hybrid-southeast-asia.yaml

damercha committed:
Update amazon-last-mile-challenges.yaml

Added several recent papers

damercha committed:
Merge branch 'main' of https://github.com/damercha/open-data-registry

damercha committed:
Fixed typo

damercha committed:
Added citation

Nakanishi-H committed:
add a first draft yaml file for draft pull request

Nakanishi-H committed:
Add one line in ADXCategories: in nifs-lhd.yaml

rileyhales committed:
Create geoglows-v2.yaml

rileyhales committed:
add hydrology tags

troyraen committed:
add satellite/mission info to descriptions

stargaser reviewed (approved):
I reviewed the updated Description sections. They all look good to me!

troyraen committed:
clarify DOIs

troyraen committed:
Merge branch 'main' into add/wise

troyraen committed:
clarify spitzer as space telescope

troyraen committed:
clarify doi

troyraen committed:
specify resource name

troyraen committed:
Merge branch 'main' into update/spitzer/seip

troyraen committed:
catalog name Database -> Table

lhuang83 committed:
Update wrf-cmip6.yaml

cstner committed:
Merge pull request #2049 from lhuang83/patch-8

Update wrf-cmip6.yaml

cstner committed:
Merge branch 'main' into update/spitzer/seip

cstner committed:
Merge pull request #2048 from troyraen/update/spitzer/seip

Update datasets/spitzer-seip

cstner committed:
Update geoglows-v2.yaml

cstner committed:
Update geoglows-v2.yaml

cstner committed:
Update geoglows-v2.yaml

cstner committed:
Update geoglows-v2.yaml

added aws-odp tag
formatting of S3 bucket, Services (list) and removed netCDF tag

cstner committed:
Merge branch 'main' into geoglowsv2

cstner committed:
Merge branch 'main' into main

cstner committed:
Update nifs-lhd.yaml

added  aws-pds tag

cstner committed:
Merge pull request #2046 from Nakanishi-H/main

add a first draft yaml file for draft pull request

rileyhales committed:
use links ending in .ipynb

cstner committed:
Merge branch 'main' into geoglowsv2

cstner committed:
Merge pull request #2047 from rileyhales/geoglowsv2

Adds GEOGloWS V2

cstner committed:
Merge branch 'main' into main

cstner committed:
Merge pull request #2045 from damercha/main

Updates to publications list

cstner committed:
Merge branch 'main' into patch-7

cstner committed:
Merge pull request #2042 from gabriellaIO/patch-7

Update io-lulc.yaml

cstner committed:
Merge branch 'main' into ncar-cesm2-arise-update

cstner committed:
Merge pull request #2036 from briandobbins/ncar-cesm2-arise-update

Added 'Explore' entry w/ URL

cstner committed:
Merge branch 'main' into add/wise

troyraen commented:
superseded by https://github.com/awslabs/open-data-registry/pull/2026
